NuGet.config

```diff
@@ -16,6 +16,8 @@
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="dotnet5-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json" />
+    <!-- Restore source for the ObjectWriter package - https://github.com/dotnet/runtimelab/issues/63 -->
```

Comment:
@MichalStrehovsky This is fixing build break in ILCompiler.csproj project restore on clean machine with empty NuGet cache.

Reply:
Hmm the CI didn't trigger in #56. Weird. Thanks for fixing this!